.github/workflows/security.yml

```diff
@@ -0,0 +1,28 @@
+on: 
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - "**/*.md"
+  schedule:
+    - cron: 15 14 * * 6
+
+jobs:
+  veracode-sca-task:
+    runs-on: ubuntu-latest
+    name: Scan repository for Issues
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Run Veracode SCA
+        env:
+          SRCCLR_API_TOKEN: ${{ secrets.SRCCLR_API_TOKEN }}
+        uses: veracode/veracode-sca@v1.09
+
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          quick: true
+          create-issues: true 
+          fail-on-cvss: 1
```

Comment:
`fail-on-cvss` is `The maximum allowed cvss in found vulnerabilities to pass the step` so should it be 0?

Reply:
You are right

Reply:
IMHO we should not fail the build. We should just create issues.

As you can see, both of these issues are not actionable by us as they are dbt dependencies.

Checking for issue title [CVE: 0000-0000 found in Jinja2 - Version: 2.11.3 [PYTHON]]
Checking for issue title [CVE: 2020-14343 found in PyYAML - Version: 5.3.1 [PYTHON]]

Reply:
`fail-on-cvss: 0` will fail the build or just create issues?

Reply:
Yes it would fail if they found one or more issues. I think we may just omit this parameter (the default is 10).